content/docs/deployment/csminstallationwizard/src/index.html

```diff
@@ -325,18 +325,10 @@
                       </div>
 
                       <div class="row mb-3">
-                        <label for="vSphere-vCenter-username" class="col-sm-2 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Username from the vCenter credentials" style="width: 160px;">vCenter UserName</label>
+                        <label for="vSphere-vCenter-cred-secret" class="col-sm-2 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Secret name for the vCenter credentials" style="width: 170px;">vCenter Cred Secret</label>
                         <div class="col-sm-9"></div>
                         <div class="was-validated col-sm-3">
-                          <input class="form-control input-lg" type="text" id="vSphere-vCenter-username" placeholder="Example: user" value="user" oninput="validateInput(validateForm, CONSTANTS)" required>
-                        </div>
-                      </div>
-                      
-                      <div class="row mb-3">
-                        <label for="vSphere-vCenter-password" class="col-sm-2 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Password from the vCenter credentials" style="width: 150px;">vCenter Password</label>
-                        <div class="col-sm-9"></div>
-                        <div class="was-validated col-sm-3">
-                          <input class="form-control input-lg" type="password" id="vSphere-vCenter-password" placeholder="Example: pwd" value="pwd" oninput="validateInput(validateForm, CONSTANTS)" required>
+                          <input class="form-control input-lg" type="text" id="vSphere-vCenter-cred-secret" placeholder="Example: vcenter-creds" value="vcenter-creds" oninput="validateInput(validateForm, CONSTANTS)" required>
```

Comment:
Should this remain as type="password" so it's hidden

Reply:
This will be just the name of the secret that would have been already created.